opent2t.p.light/opent2t.p.light.raml

```diff
@@ -0,0 +1,295 @@
+                    "n": "Device 1",
+                    "di": "A94F0FBE-9D2D-4A81-A99B-4024AADB1631",
+                    "icv": "core.1.1.0",
+                    "dmv": "res.1.1.0",
+                    "resources": [
+                      {
+                        "href": "/power",
+                        "rt": ["oic.r.switch.binary"],
+                        "if": ["oic.if.a", "oic.if.baseline"]
+                      },
+                      {
+                        "href": "/colourrgb",
+                        "rt": ["oic.r.colourrgb"],
+                        "if": ["oic.if.a", "oic.if.baseline"]
+                      },
+                      {
+                        "href": "/dimming",
+                        "rt": ["oic.r.dimming"],
+                        "if": ["oic.if.a", "oic.if.baseline"]
+                      }
+                    ]
+                  }
+                ]
+              }
+
+/?$expand
+  displayName: Platform
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Lamp
+            example: |
+              {
+                "opent2t": {
+                  "schema": "opent2t.p.outlet",
+                  "translator": "opent2t-translator-com-opent2t-p-lamp@0.1.1",
+                  "control_id": "123"
+                },
+                "pi": "8045C566-B44B-462F-A4AB-FA7FC968CE71",
+                "mnmn": "Contoso",
+                "n": "Bedside lamp",
+                "rt": ["opent2t.p.lamp"],
+                "entities": [
+                  {
+                    "rt": ["oic.d.lamp"],
+                    "n": "Device 1",
+                    "di": "A94F0FBE-9D2D-4A81-A99B-4024AADB1631",
+                    "icv": "core.1.1.0",
+                    "dmv": "res.1.1.0",
+                    "resources": [
+                      {
+                        "href": "/power",
+                        "rt": ["oic.r.switch.binary"],
+                        "if": ["oic.if.a", "oic.if.baseline"],
+                        "id": "power",
+                        "value": true
+                      },
+                      {
+                        "href": "/colourrgb",
+                        "rt": ["oic.r.colour.rgb"],
+                        "if": ["oic.if.a", "oic.if.baseline"],
+                        "id": colour,
+                        "rgbValue": [255,255,255],
+                        "range": [0,255]
+                      },
+                      {
+                        "href": "/dimming",
+                        "rt": ["oic.r.light.dimming"],
+                        "if": ["oic.if.a", "oic.if.baseline"]
+                        "id": "dimming",
+                        "dimmingSetting": 30,
+                        "step": 5,
+                        "range": [0, 100]
+                      }
+                    ]
+                  }
+                ]
+              }
+
+/device/{deviceid]/power:
```

Comment:
Each of these resources can be exposed relative to a single /devices/{deviceid} so we don't have to duplicate the full path every time, e.g.:

```raml
/devices/{deviceid}:
  /power:
    displayName: Power
    ...

  /dim:
    displayName: Dim
    ...
```

Also note: There's a typo in all of the resource paths. You have a ] instead of }